presto-rcfile/src/test/java/com/facebook/presto/rcfile/AbstractTestRcFileReader.java

```diff
@@ -159,7 +160,14 @@ public void testTimestampSequence()
                 TIMESTAMP,
                 intsBetween(-31_234, 31_234).stream()
                         .filter(i -> i % 19 == 0)
-                        .map(timestamp -> new SqlTimestamp(timestamp, UTC_KEY))
+                        .map(timestamp -> {
+                            if (SESSION.isLegacyTimestamp()) {
```

Comment:
You know `SESSION`  is has `UTC` time zone key, so you could simply `sqlTimestampOf(timestamp, SESSION)`, right?